src/main/kotlin/com/hedvig/rapio/exceptionhandler/RestExceptionHandler.kt

```diff
@@ -68,6 +70,15 @@ class RestExceptionHandler : ResponseEntityExceptionHandler() {
         return ResponseEntity.status(HttpStatus.FORBIDDEN).body(ExternalErrorResponseDTO(e.message ?: ""))
```

Comment:
Do we want `""` or `null`? I suggest we eitehr change the old ones or use `""` on the new one :) 
```suggestion
        return ResponseEntity.status(HttpStatus.FORBIDDEN).body(ExternalErrorResponseDTO(e.message))
```

Reply:
It's because message is not nullable on `MissingServletRequestParameterException`, but it is on the others 🙂